testsuite/layers/pom.xml

```diff
@@ -2266,6 +2302,7 @@
                                                 <layer>logging</layer>
                                                 <layer>management</layer>
                                                 <layer>management-security-realm</layer>
+                                                <layer>microprofile-platform</layer>
                                                 <layer>microprofile-config</layer>
```

Comment:
@rhusar given `microprofile-platform` already includes `microprofile-config`, `microprofile-fault-tolerance`, `microprofile-health`, `microprofile-jwt`, `microprofile-metrics`, `microprofile-openapi`, `open-tracing`: can we remove them from the list? or we keep them to prevent errors in case, in some future, they are removed from `microprofile-platform` or `microprofile-platform` itself is removed?

Reply:
@tommaso-borgato We can – but the existing layers are already overlapping (e.g. undertow is listed explicitly even though its a dependency of any of the web server layers, e.g. datasources-web-server). So this is intentional.